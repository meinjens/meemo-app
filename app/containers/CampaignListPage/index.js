import React from 'react';

export default function FeaturePage() {
  return (
    <div>
      <p>Test</p>
    </div>
  );
}
