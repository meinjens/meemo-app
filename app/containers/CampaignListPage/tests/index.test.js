import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import CampaignListPage from '../index';

describe('<CampaignListPage />', () => {
  it('should render its heading', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <CampaignListPage />
      </IntlProvider>,
    );

    expect(firstChild).toMatchSnapshot();
  });
});
