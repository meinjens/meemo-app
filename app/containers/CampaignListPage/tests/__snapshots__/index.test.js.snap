// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CampaignListPage /> should render its heading 1`] = `
<div>
  <p>
    Test
  </p>
</div>
`;
